packages/tse-backend/src/common/interfaces/exchange-operation.interfaces.ts

```diff
@@ -0,0 +1,13 @@
+import { Order } from '../entities/order.entity';
+import { OrderStatus } from '../enums/exchange-operation.enums';
+
+export interface SaveOperationDto {
+  status: OrderStatus;
+  details: Record<string, any>;
+  order: Order;
+}
+
+export interface PersistOperationDto {
```

Comment:
What's the difference in `Persist` vs `Save`?

Reply:
`Persist` is used in the service, while `Save` is used in the repository.
```
  async persistOperationData(order: Order, data: PersistOperationDto) {
    await this.repository.save({
      status: data.status,
      details: data.details,
      order,
    } as SaveOperationDto);
  }
```